adm/style/manage_pages.html

```diff
@@ -21,7 +21,7 @@
 
 		<p>{L_ACP_PAGES_CREATE_PAGE_EXPLAIN}</p>
 	<!-- ELSE -->
-		<h1>{L_ACP_PAGES_EDIT_PAGE} {L_COLON}{L_COLON} {PAGES_TITLE}</h1>
+		<h1>{L_ACP_PAGES_EDIT_PAGE} :: {PAGES_TITLE}</h1>
```

Comment:
Requested by @nickvergessen... they're used for stylization here rather than actual grammar.